.vib/owncloud/cypress/cypress/fixtures/links.json

```diff
@@ -0,0 +1,5 @@
+{
+  "newLinks": {
```

Comment:
Minor style comment, this should be in the singular tense.

Reply:
If I recall correctly, @joancafom and I agreed on a different PR to use the plural for this even though we have only one item due to consistency. Is my memory correct @joancafom?

Reply:
The plural is for the file name, `links`, as it might potentially have more than one link. If the item inside only contains a single link, I'd go with singular as well.